app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __str__(self) -> str:
+        return "Not vaccinated"
```

Comment:
Consider providing a more descriptive error message, such as `return 'Visitor is not vaccinated'`.